src/partition/ReceivedPartition.cpp

```diff
@@ -124,6 +126,14 @@ void ReceivedPartition::compute()
   if (_toMapping)
     _toMapping->tagMeshFirstRound();
 
+  // if(utils::MasterSlave::getRank()==0)
+  // {
+  //   for (auto & vtx : _mesh->vertices())
+  //   {
+  //     std::cout<< vtx.getGlobalIndex() << std::endl;
+  //   }
+  // }
```

Comment:
Let's not push this to develop. If you think this is helpful in the long run you could make it a logging output and potentially guard with `#ifndef NDEBUG`.